lighthouse-core/audits/byte-efficiency/unused-javascript.js

```diff
@@ -79,15 +79,18 @@ class UnusedJavaScript extends ByteEfficiencyAudit {
    * @return {Promise<ByteEfficiencyAudit.ByteEfficiencyProduct>}
    */
   static async audit_(artifacts, networkRecords, context) {
-    const bundles = await JsBundles.request(artifacts, context);
+    let bundles;
+    if (artifacts.SourceMaps) {
```

Comment:
While we're FWIWing...

FWIW I liked your elegant solution from before, just a 1-line change too :)